stdlib/Sockets/src/Sockets.jl

```diff
@@ -478,7 +478,11 @@ Enables or disables Nagle's algorithm on a given TCP server or socket.
 """
 function nagle(sock::Union{TCPServer, TCPSocket}, enable::Bool)
     # disable or enable Nagle's algorithm on all OSes
-    ccall(:uv_tcp_nodelay, Cint, (Ptr{Cvoid}, Cint), sock.handle, Cint(!enable))
+    r = ccall(:uv_tcp_nodelay, Cint, (Ptr{Cvoid}, Cint), sock.handle, Cint(!enable))
+    if r < 0
+        enable_disable = ifelse(enable, "enable", "disable")
+        @warn "Cannot $(enable_disable) Nagle's algorithm ( Error: $(Libc.strerror(Libc.errno())) )" maxlog=1
```

Comment:
Having spaces inside of parens like this is not standard in Julia error messages. Maybe have the error message on a new line?

Reply:
It would be more standard now to use the exception kwarg

Reply:
I've changed it, the error is now printed using the exception kwarg.